rskj-core/src/main/java/co/rsk/peg/federation/FederationStorageProviderImpl.java

```diff
@@ -385,6 +398,18 @@ private void saveProposedFederation(ActivationConfig.ForBlock activations) {
         bridgeStorageAccessor.saveToRepository(PROPOSED_FEDERATION.getKey(), proposedFederation, BridgeSerializationUtils::serializeFederation);
     }
 
+    private void saveFundTransactionUnsignedHash(ActivationConfig.ForBlock activations) {
+
+        if (!activations.isActive(RSKIP419) || !isFundTransactionUnsignedHashSet) {
+            return;
+        }
+
+        byte[] data = nonNull(fundTransactionUnsignedHash) ? fundTransactionUnsignedHash.getBytes() : null;
```

Comment:
Is this the proper way to serialize a Sha256 hash?

Reply:
Updated.